mobile/android/android-emulator-services.ts

```diff
@@ -327,10 +327,9 @@ class AndroidEmulatorServices implements Mobile.IEmulatorPlatformServices {
 		return (() => {
 			var minVersion = this.$emulatorSettingsService.minVersion;
 
-			var best =_(this.getAvds().wait())
+			var best = _.chain(this.getAvds().wait())
```

Comment:
Yes, because there's problem with lodash definitions. After we fix them, _.chain will not be needed.